gitlab/mixins.py

```diff
@@ -223,6 +223,18 @@ def get_update_attrs(self):
         """
         return getattr(self, '_update_attrs', (tuple(), tuple()))
 
+    def get_update_method(self):
```

Comment:
Please make this method private (`_get_update_method`), I don't see any reason to expose it to the user.

Reply:
Done.